plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlClient.java

```diff
@@ -316,6 +320,14 @@ protected String getTableSchemaName(ResultSet resultSet)
         return legacyToPrestoType(session, connection, typeHandle);
     }
 
+    private static Optional<ColumnMapping> getVarcharType(int varcharLength)
```

Comment:
The method does not return a varchar type, as the name suggests.
Maybe `varcharColumnMapping`?